AtividadeSemana4_Lorena/Exercicio36_Tabuada_Lorena.py

```diff
@@ -0,0 +1,10 @@
+base_tabuada = int(input("\nInsira um número para obter a tabuada deste número: "))
+inicio_tabuada = int(input("Insira um número inicial para a tabuada: "))
+fim_tabuada = int(input("Insira um número final para a tabuada: ")) + 1
+
+if inicio_tabuada <= fim_tabuada:
```

Comment:
Aqui tem um pequeno errinho lógico. 
Como você está adicionando um à variável fim_tabuada no momento em que a variável é recebida, se o usuário pedir uma tabuada de n até n-1, a condição será verdadeira e o código não vai cair no else, mas o range ficará vazio.

 Por exemplo, se o usuário pedir a tabuada do 5, começando em 4 e terminando em 3, sua condição se tornará 
 `if 4 <= 4`, mas seu range terá 0 items e o for não será iterado nenhuma vez.  
 
 
![image](https://github.com/reprograma/on29-python-s04-logica-III/assets/40744589/807eead3-797e-4303-8636-8831aeeceada)